importers/dependencies/extract_relations.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from importers.dependencies.languages import Parser
+from util.deps_util import DependencyUtils
+from util.logging_util import LoggingUtil
+import os
+
+
+class DependencyExtractor(object):
```

Comment:
yes, good idea! ref_id is a better name, since the gitana enduser may want to analyse also a tag.